### DISCUSSION TITLE
Remove the background from hipattern

### DISCUSSION THREAD

**Discussion by Ballpin** [nvim-mini/mini.nvim#1323] [Q&A] [answered]
In lazyvim there is this configuration https://www.lazyvim.org/extras/util/mini-hipatterns any idea how i can remove the background and only have the fg set?

I tried change bg to fg but no luck :(

**Comment by echasnovski** [comment, marked as answer]
I think this is more of a question to LazyVim. As far as I understand, using its "Full Spec" code instead of using an extra directly *and* making appropriate changes to its code (such as replacing `"bg"` with `"fg"`) should work.
